service/debugger/debugger.go

```diff
@@ -1310,6 +1310,23 @@ func (d *Debugger) ListDynamicLibraries() []api.Image {
 	return r
 }
 
+// ExamineMemory returns a list of specific address memory that len is num.
```

Comment:
This comment should be rewritten to something like: `ExamineMemory returns the raw memory stored at the given address. The amount of data to be read is specified by `length`. This function will return an error if it reads less than ``length`` bytes.`